.github/actions/docs/entrypoint.sh

```diff
@@ -3,6 +3,7 @@
 apt-get update
 apt-get install -y git
 git reset --hard HEAD
+git fetch origin
```

Comment:
Maybe its better to fetch origin before resetting to HEAD:
```suggestion
git fetch origin
git reset --hard HEAD
```